sandbox/darren/file_search.py

```diff
@@ -58,7 +58,7 @@ def on_mount(self) -> None:
         self.mount(file_table_wrapper=Widget(self.file_table))
         self.mount(search_bar=self.search_bar)
 
-    def handle_changed(self, event: TextWidgetBase.Changed) -> None:
+    def on_text_widget_base_changed(self, event: TextWidgetBase.Changed) -> None:
```

Comment:
We might want to explicitly set the namespace here. Reads a little clumsily. Guess we could set the namespace to "text_widget" so the method would be "on_text_widget_changed".